File: bench/perf/js/wordcount.js
const seed = "the quick brown fox jumps over the lazy dog ";
const text = seed.repeat(500000);
const words = text.split(" ");
const counts = new Map();
for (const w of words) counts.set(w, (counts.get(w) || 0) + 1);
console.log(counts.size);
console.log(counts.get("the"));

File: bench/perf/rust/wordcount.rs
use std::collections::HashMap;
fn main() {
    let seed = "the quick brown fox jumps over the lazy dog ";
    let text = seed.repeat(500000);
    let words: Vec<&str> = text.split(' ').collect();
    let mut counts: HashMap<&str, i64> = HashMap::new();
    for w in &words {
        *counts.entry(*w).or_insert(0) += 1;
    }
    println!("{}", counts.len());
    println!("{}", counts["the"]);
}
